objects/obj_drop_select/Alarm_5.gml

```diff
@@ -57,35 +77,34 @@ argument0.p_chaos[argument1]=4;
 argument0.p_owner[argument1]=10;
 */
 
-var co, v, stop;
-co=0;v=0;stop=0;  
-repeat(3600){
-    if (co<11){v+=1;
-        if (v>300){co+=1;v=1;}
+var co=0,v=0,stop=0;
```

Comment:
could you rename these? also stop = false (if it's a boolean)

Reply:
renamed co as company (since it refers up to 10 including 0 as HQ which is what im guessing) and stop to true and false. Do you know what v is representing? I could chane it to var or value but nothing which would help comprehend the code

Reply:
v looks like current unit limit for a company, so after v = 300, we assume that it's time to switch to next company?

Reply:
yes, probably the hard cap for max units (including vehicles?)